### ISSUE TITLE
Add media play buttons to title bars.

### ISSUE THREAD

**Issue by bluemarvin** [MozillaReality/FirefoxReality#1651]
**Is your feature request related to a problem? Please describe.**
Users should be able to pause video and audio in a window that does not have focus.

**Describe the solution you'd like**
We should put play/pause buttons in the title bar when the window does not have focus so the user can pause/play with out giving the window focus.

**Comment by bluemarvin** [comment]
@thenadj I think this was dropped from the title bar spec?

**Comment by thenadj** [comment]
@bluemarvin Good catch! I'll create this and will post here when it's done. Is this a 1.4 feature?

**Comment by jvonitter** [comment]
@thenadj preferably yes, if we can get it done quickly!

**Comment by thenadj** [comment]
Attached is the updated UIS plus the two icons for the button. 
[pause-play_icons.zip](https://github.com/MozillaReality/FirefoxReality/files/3536796/pause-play_icons.zip)
[UIS-69 Window Title_007.pdf](https://github.com/MozillaReality/FirefoxReality/files/3536797/UIS-69.Window.Title_007.pdf)

**Comment by jvonitter** [comment]
@thenadj you are awesome. Thanks for being so speedy!
